examples/installation/build.sh

```diff
@@ -106,7 +106,7 @@ xctest() {
         )
     elif [[ $NAME == SwiftPackageManager* ]]; then
         if [ -n "$sha" ]; then
-            sed -i '' 's@branch = "master"@branch = "'"$sha"'"@' "$DIRECTORY/$NAME.xcodeproj/project.pbxproj"
+            ex '+%s@branch = "master"@branch = "'"$sha"'"@' -scwq "$DIRECTORY/$NAME.xcodeproj/project.pbxproj"
```

Comment:
`sed -i` doesn't work on symlinks and this file has been a symlink for a while. `ex` (the line-editor part of vim) does and is a reasonably drop-in replacement.